examples/98-common/create-dimension-line.py

```diff
@@ -0,0 +1,86 @@
+"""
+Create Dimension Line
+---------------------
+
+Create a 2d dimension line along 2d structured mesh.
+
+"""
+
+import numpy as np
+
+import pyvista as pv
+
+# Define x and y ranges for the structured mesh
+xrng = np.arange(-10, 10, 2)
+yrng = np.arange(-10, 10, 5)
+
+# Create the structured mesh
+grid = pv.RectilinearGrid(xrng, yrng)
+
+
+###############################################################################
+# Plot mesh with dimension line
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+# Create a plotter object
+plotter = pv.Plotter()
+
+# Add the mesh to the plotter object
+plotter.add_mesh(grid, show_edges=True, color='tan')
+
+# Enable parallel projection for the plot
+plotter.enable_parallel_projection()
+
+# If pointa and pointb are specified, the dimension between them is drawn.
```

Comment:
We must specify `pointa` and `pointb`.  I gave a suggestion, but I'm not sure if hte style is double `` or single `
```suggestion
# The dimension between ``pointa`` and ``pointb`` is drawn.
```